README.md

```diff
@@ -32,16 +32,14 @@ configuration.
 
     git clone https://github.com/opendatateam/udata-id.git
 
-The project being used by _uData_, ensure that the _udata\_id_ package is
-available from your _udata_ project's _virtualenv_ (eg: `add2virtualenv /path/to/my/udata-id`).
+Get into your **_udata_** project folder, activate its virtualenv, and install
+the _udata-id_ from there:
 
```

Comment:
It is a detail, but there is no real need to go the udata project folder. The point is to activate the udata venv, which can be done from anywhere.